docs/content/commands.md

```diff
@@ -117,33 +123,35 @@ public class BuildWallHandlerSystem : ComponentSystem
         for(var i = 0; i < data.Length; i++)
         {
             var requests = data.BuildWallRequests[i];
+            var responder = data.BuildWallResponders[i];
 
-            foreach (var request in requests.Buffer)
+            foreach (var request in requests.Requests)
             {
                 // Do something with the request
                 var buildRequest = request.Request;
                 
-                BuildResponse buildResponse = new BuildResponse
-                {
-                    ...
-                }
+                Builder.BuildWall.Response buildResponse = Builder.BuildWall.Response.CreateResponse
+                (
+                    buildRequest,
+                    new BuildResponse(...)
+                );
                 
-                request.SendBuildWallResponse(buildResponse);
+                responder.Response.Add(buildResponse);
             }
         }
     }
 }
 ```
 
-`build_wall` command requests are on a `CommandRequests<BuildWall.Request>` ECS component. The worker instance should respond to the request using the `SendBuildWallResponse(BuildResponse buildResponse)` method on the request itself.
+`build_wall` command requests are on a `Builder.CommandRequest.BuildWall` ECS component. The worker instance should respond to the request using the `Builder.CommandResponders.BuildWall` ECS component.
 
 ### Receiving command responses
 
-Like requests, when an ECS entity receives a command response, the Unity GDK attaches a `CommandResponses<T>` ECS component to the ECS entity (where `T` implements `IIncomingCommandResponse`).
+Like requests, when an ECS entity receives a command response, the Unity GDK attaches a `ComponentName.CommandResponses.CommandName` ECS component to the ECS entity, where `ComponentName` is the SpatialOS component the command is defined in and `CommandName` is the name of the command in schema.
 
-The ECS entity that sent the request receives the response. The response object includes the payload of the response and the request payload that originally send the command. **This payload is null** when no payload is sent back.
+`ComponentName.CommandResponses.CommandName` contains a list of `ComponentName.CommandName.ReceivedResponse`. The `ComponentName.CommandName.ReceivedResponse` includes the payload of the response and the request payload that originally was sent with the command. **This payload is null** when the command fails.
 
-> **Note**: `CommandResponses<T>` is a `Component`, not an `IComponentData`. This means you must use a `ComponentArray<T>` for injection rather than a `ComponentDataArray<T>`.
+The ECS entity that sent the request receives the response. The response object.
```

Comment:
Not sure what "The response object." means here?

Reply:
Was erroneous. Removed